src/qt/optionsdialog.h

```diff
@@ -50,7 +56,7 @@ private Q_SLOTS:
     void on_openBitcoinConfButton_clicked();
     void on_okButton_clicked();
     void on_cancelButton_clicked();
-    
```

Comment:
nit, unnecessary change.

Reply:
Could you remove this change to avoid conflict with #13753.

Reply:
I removed it, even though intentionally adding whitespaces (the ones I removed) caused a bit of sadness inside. Even if it was for a good cause.

There was another instance, I removed that as well.

At the end of the day, all trailing whitespaces will be removed which gives me a warm fuzzy feeling.

Thx for the reviews btw